acceptance/acceptance_test.go

```diff
@@ -739,6 +738,7 @@ func testAcceptance(
 							runImageMirror,
 						)
 						assert.Nil(err)
+
```

Comment:
nit: Remove
(Totally no need to do that if there's no need for a further commit, just a very very very tiny thing that caught my eye at the top)

Reply:
Ya this paired with the last extra function I missed in the `acceptance_test.go` file make me think they should be linted during `make verify`.